jujugui/static/gui/src/app/components/entity-content/entity-content.js

```diff
@@ -150,7 +150,7 @@ YUI.add('entity-content', function() {
         var submitLink = 'https://bugs.launchpad.net/charms/+source/' +
           `${entityModel.get('name')}/+filebug`;
         var contributeLink = 'https://code.launchpad.net/~charmers/charms/' +
-          `trusty/${entityModel.get('name')}/trunk`;
+          `${entityModel.get('series')}/${entityModel.get('name')}/trunk`;
```

Comment:
Ah, as you've noted below, the URL doesn't seem to work for all bundles. Not sure what we should do here...